integration/runIntegration.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+cd ..
+make env_up
+docker run --rm --network neo_go_network -p 20331-20332:20331-20332 -p 2112:2112 --name neo-go-integration --ip 172.200.0.5 -t nspccdev/neo-go-integration &
```

Comment:
You may replace `docker run & sleep` with docker-compose file, thus script would look easier to read. For instance:

```
version: "2.4"

services:
  neo_go_integration:
    container_name: neo-go-integration
    image: nspccdev/neo-go-integration
    tty: true
    ports:
      - "20331-20332:20331-20332"
      - "2112:2112"
    networks:
      neo_go_network:
        ipv4_address: "172.200.0.5"

networks:
  neo_go_network:
    name: neo_go_network
    driver: bridge  
```

Reply:
but anyway we would need to sleep to wait until neo-go-integration connects to privatenet

Reply:
and with docker compose there is one issue: once you want to stop it it will remove network and after that `make env_down` won't run since network couldn't removed(it's in use by privatenet)